iroha_network/src/lib.rs

```diff
@@ -18,7 +18,7 @@ pub mod prelude {
     pub use crate::{AsyncStream, Network, Request, Response, State};
 }
 
-pub const BUFFER_SIZE: usize = 2048;
+pub const BUFFER_SIZE: usize = 1024;
```

Comment:
What is the purpose of this change? I am not against, just asking :)

Reply:
Want make our constraints stricter.